src/components/input/input.js

```diff
@@ -135,6 +135,7 @@ function labelDirective() {
  * @param {string=} placeholder An alternative approach to using aria-label when the label is not
  *   PRESENT. The placeholder text is copied to the aria-label attribute.
  * @param md-no-autogrow {boolean=} When present, textareas will not grow automatically.
+ * @param md-no-asterisk {boolean=} Stops appending of an asterisk to the label of required inputs
```

Comment:
I'd change this to 
`When present, asterisk will not be appended to required inputs label`
or something like that